setup.py

```diff
@@ -366,7 +366,7 @@ def run_tests(self):
         package_dir=package_dir,
         package_data=package_data,
         classifiers=classifiers,
-        download_url="https://downloads.sourceforge.net/project/matplotlib/matplotlib/matplotlib-{0}/matplotlib-{0}.tar.gz".format(__version__),
+        download_url="http://matplotlib.org/users/installing.html",
```

Comment:
Apparently it doesn't have to be, and it's used for information only.  Numpy and Scipy both do something similar: just pointing to a directory page of files.